### PR TITLE
docs: add nuxt.js

### PR DESCRIPTION
**Pull request by neo773**
## Summary
This PR adds instrumentation documentation for Nuxt.js

## Are there any deployment considerations?
No.

## Does this work require review from our design team?
No.

### CODE REVIEW

**Comment by neo773** [comment]
@Vadman97 Could I get a review on this?

File: docs-content/getting-started/fullstack-frameworks/nuxt.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+title: Nuxt.js Walkthrough
+slug: nuxt
+heading: Nuxt.js Walkthrough
+createdAt: 2024-08-13T18:48:40.767Z
+updatedAt: 2024-08-13T18:48:40.767Z
+---
+
+## Overview
+
+Our Node.js SDK integrates seamlessly with Nuxt.js, providing both frontend session replays and server-side monitoring capabilities.
```
Comment (Vadman97) [inline docs-content/getting-started/fullstack-frameworks/nuxt.md:11]: add link to https://nuxt.com/

File: docs-content/getting-started/fullstack-frameworks/nuxt.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		H.init(highlightConfig)
+	}
+
+	nitro.hooks.hook('error', async (error, { event }) => {
+		const headers = event?.node.req.headers!
+		const parsed = H.parseHeaders(headers)
+		if (parsed !== undefined) {
+			H.consumeError(error, parsed?.secureSessionId, parsed?.requestId)
+		}
+	})
```
Comment (Vadman97) [inline docs-content/getting-started/fullstack-frameworks/nuxt.md:42]: any other hooks that could be used to `H.startActiveSpan`?

**Review by Vadman97 — APPROVED** [review]
thanks for your contribution! requesting a few small edits, but should be generally good to go

**Comment by neo773** [comment]
@Vadman97 
Pushed!

**Comment by Vadman97** [comment]
@neo773 could you please update on our repo as we've pushed some fixes for open source contributors' CI

**Comment by neo773** [comment]
@Vadman97 
I just updated the branch but it's still failing 
image

**Comment by neo773** [comment]
@Vadman97 
Can we get this merged?

**Comment by neo773** [comment]
> @Vadman97 Can we get this merged?

@Vadman97 
Hi,

Just following up in case my previous message missed your inbox.

File: docs-content/getting-started/fullstack-frameworks/nuxt.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	nitro.hooks.hook('afterResponse', () => {
+		H.endActiveSpans()
+	})
+
+	nitro.hooks.hook('error', async (error, { event }) => {
+		const headers = event?.node.req.headers!
+		const parsed = H.parseHeaders(headers)
+		if (parsed !== undefined) {
+			H.consumeError(error, parsed?.secureSessionId, parsed?.requestId)
+		}
```
Comment (anbraten) [inline docs-content/getting-started/fullstack-frameworks/nuxt.md:56]: This allows tracking all nuxt errors. Those related to a frontend session and internal ones.

```suggestion
		const headers = event?.node.req.headers
		const parsedHeaders = headers ? H.parseHeaders(headers) : undefined
		H.consumeError(error, parsedHeaders?.secureSessionId, parsedHeaders?.requestId)
```

Comment (anbraten) [inline docs-content/getting-started/fullstack-frameworks/nuxt.md:56]: This allows tracking all nuxt errors. Those related to a frontend session and internal ones.

```suggestion
		const headers = event?.node.req.headers
		const parsedHeaders = headers ? H.parseHeaders(headers) : undefined
		H.consumeError(error, parsedHeaders?.secureSessionId, parsedHeaders?.requestId)
```

**Comment by anbraten** [comment]
As nuxt is also the "frontend" / client part it would be great if the usage of the client sdk with nuxt could also be explained similar to: https://www.highlight.io/docs/getting-started/client-sdk/vuejs

**Pull request closed without merging**